distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java

```diff
@@ -32,14 +59,38 @@
 
     private static final String CATALOG_REGISTRY = "catalog-registry";
 
+    private static final String REGISTRY_CATALOG_STORE_ID = "cswRegistryCatalogStore";
+
     @BeforeExam
     public void beforeExam() throws Exception {
         try {
             basePort = getBasePort();
             getAdminConfig().setLogLevels();
             getServiceManager().waitForRequiredApps(getDefaultRequiredApps());
+            getServiceManager().startFeature(true, CATALOG_REGISTRY);
             getServiceManager().waitForAllBundles();
             getCatalogBundle().waitForCatalogProvider();
+            getServiceManager().waitForHttpEndpoint(SERVICE_ROOT + "/catalog/query?_wadl");
+            CswRegistryStoreProperties cswRegistryStoreProperties = new CswRegistryStoreProperties(
+                    REGISTRY_CATALOG_STORE_ID);
+            getServiceManager().createManagedService(cswRegistryStoreProperties.FACTORY_PID,
+                    cswRegistryStoreProperties);
+            getCatalogBundle().waitForCatalogStore(REGISTRY_CATALOG_STORE_ID);
+
+            MetacardImpl metacard = new MetacardImpl(new RegistryObjectMetacardType());
```

Comment:
The reason we are doing the tests this way and not through the csw endpoint is because there is currently no way to specify destinations in the request which we need for testing. We talked to phil about it and this is what he recommended.